app/autodiscovery/src/main/java/com/fsck/k9/autodiscovery/srvrecords/SrvServiceDiscovery.kt

```diff
@@ -0,0 +1,96 @@
+package com.fsck.k9.autodiscovery.srvrecords
+
+import com.fsck.k9.autodiscovery.ConnectionSettings
+import com.fsck.k9.autodiscovery.ConnectionSettingsDiscovery
+import com.fsck.k9.helper.EmailHelper
+import com.fsck.k9.mail.AuthType
+import com.fsck.k9.mail.ConnectionSecurity
+import com.fsck.k9.mail.ServerSettings
+import java.util.Locale
+import org.minidns.dnsname.DnsName
+import org.minidns.hla.ResolverApi
+import org.minidns.hla.SrvProto
+
+enum class SrvType(var protocol: String, var assumeTls: Boolean) {
+    SUBMISSION("smtp", false),
+    IMAPS("imap", true),
+    POP3S("pop", true),
```

Comment:
We want to remove POP3 support in the future. The first step towards this is not supporting auto-setup for it. All POP3 settings have been removed from `providers.xml` already. So this `ConnectionSettingsDiscovery` implementation doesn't need to support POP3 either.